src/tools/epics.ts

```diff
@@ -61,6 +62,17 @@ export class EpicTools extends BaseTools {
 			async (params) => await tools.searchEpics(params),
 		);
 
+		server.tool(
+			"create-epic",
+			"Create a new Shortcut epic.",
+			{
+				groupId: z.string().describe("The ID of the group or team to assign the epic to"),
```

Comment:
This is a problem with a rename in Shortcut some years back. We originally had "groups" but decided to rename them to "teams". But we still have the concept of "groups" in the code. To avoid confusing the AI, we refer to it as `team` everywhere in this MCP. That way the terminology the user would use match the terminology in the code.

```suggestion
				teamId: z.string().describe("The ID of the team to assign the epic to"),
```